packages/extension-huggingface/src/index.ts

```diff
@@ -0,0 +1,10 @@
+// Alias the Builder and Runner from the package and export them to prevent Extension loader loading the same Builder and Runner class when creating multiple Functional Filters or Tags.
+import {
+  Builder as HuggingFaceTableQuestionAnsweringFilterBuilder,
+  Runner as HuggingFaceTableQuestionAnsweringFilterRunner,
+} from './lib/tableQuestionAnswering';
+
```

Comment:
i think the extension name "extension-filter-huggingface" will be more clear than "extension-huggingface"

Reply:
Thanks @onlyjackfrost for the suggestion, after the discussion, we keep the original `extension-huggingface` naming, because the hugging face may not only implemented by filters, but also tags in the future.